app/views/UpdateCaseConfirmationView.scala.html

```diff
@@ -43,13 +43,18 @@ <h1 class="govuk-panel__title">
 
     @p {
         @Html(messages("updateCase.confirmation.whatYouShouldDoNext.p"))
-        <a class="govuk-link" href="mailto:customsaccountingrepayments@@hmrc.gov.uk">customsaccountingrepayments@@hmrc.gov.uk</a>.
+        <a class="govuk-link" href="mailto:customsaccountingrepayments@@hmrc.gov.uk">
+            customsaccountingrepayments@@hmrc.gov.uk</a>.
     }
 
     <div id="helpImproveServiceLink">
-        @p{
-            <a href=@{appConfig.surveyUrl} class="govuk-link">@{messages("updateCase.confirmation.helpImproveServiceLink")}</a>
-        }
+    @p {
+        <a href=@{
```

Comment:
breaking up the view elements like this doesn't seem to fit with how I would expect to see html elements